izadji/src/components/TeamMemberItem/TeamMemberItem.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import styles from './TeamMemberItem.module.scss';
+
+export type TeamMemberItemProps = {
+  name: string;
+  jobTitle: string;
+  profileImage: string;
+  email: string;
+  bio: string;
```

Comment:
It seems to me that we do not use **bio** prop anywhere inside the TeamMemberItem component. Maybe we can delete it?

Reply:
:)